Add informations about the mail() method not being used anymore (#820)
PrestaShop/docs: src/content/1.7/modules/core-updates/1.7.7.md

Also new in this version, when a module is enabled/disabled all its related [Tab links are automatically shown/hidden]({{< ref "1.7/modules/concepts/controllers/admin-controllers/tabs#automatic-hiding-of-disabled-modules" >}}).

## Default mail method does not use the `mail()` function anymore

The `mail()` function is not used anymore [for security reasons](https://github.com/swiftmailer/swiftmailer/issues/866#issuecomment-289291228).

> Instead, `/usr/sbin/sendmail` took the default way of sending mails.

Reviewer: ```suggestion
Instead, `/usr/sbin/sendmail` is now the default way of sending mails.
```

{{% notice warning %}}

> You should check that you environment is configured to allow this use or change this settings in **Configure** > **Advanced Parameters** > **E-mail**.

Reviewer: ```suggestion
You should check that you environment is configured to allow this use or change this setting in **Configure** > **Advanced Parameters** > **E-mail** by another sending method like SMTP.
```

> You should check that you environment is configured to allow this use or change this settings in **Configure** > **Advanced Parameters** > **E-mail**.

Reviewer: ```suggestion
You should check that your server environment is configured to allow this use or change this settings in **Configure** > **Advanced Parameters** > **E-mail**.
```

> You should check that you environment is configured to allow this use or change this setting in **Configure** > **Advanced Parameters** > **E-mail** by another sending method like SMTP.

Reviewer: ```suggestion
You should check that your server environment is configured to allow this use or change this setting in **Configure** > **Advanced Parameters** > **E-mail** by another sending method like SMTP.
```

> You should check that you environment is configured to allow this use or change this setting in **Configure** > **Advanced Parameters** > **E-mail** by another sending method like SMTP.

Author: ```suggestion
You should check that your server environment is configured to allow this use or change this setting in **Configure** > **Advanced Parameters** > **E-mail** by another sending method like SMTP.
```
